tests/integration/redis-cli.tcl

```diff
@@ -808,4 +808,25 @@ if {!$::tls} { ;# fake_redis_node doesn't support TLS
         assert_equal 3 [exec {*}$cmdline ZCARD new_zset]
         assert_equal "a\n1\nb\n2\nc\n3" [exec {*}$cmdline ZRANGE new_zset 0 -1 WITHSCORES]
     }
+
+    tags {"external:skip"} {
+        test_interactive_cli_with_prompt "db_num showed in redis-cli after reconnected" {
```

Comment:
please move this test to similar block, like `test_interactive_cli_with_prompt "should disable and persist line and move the cursor if user presses tab"`.